src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Kindly make the app component class-based as mentioned in the requirements 👍

Reply:
I do not know how I missed this, thanks sir for raising my attention to the issue.